tests/network/test_services.py

```diff
@@ -105,7 +105,7 @@ def test_get_balance_main_failure(self):
             mock_network_api_main.get_balance(MAIN_ADDRESS_USED1)
 
     def test_get_transactions_main_equal(self):
-        results = [api.get_transactions(MAIN_ADDRESS_USED1) for api in network_api_main.list_of_apis]
+        results = [[*api.get_transactions(MAIN_ADDRESS_USED1)] for api in network_api_main.list_of_apis]
         assert all_items_common(results[:100])
```

Comment:
Again... I believe the reason you have added the * symbol here is because adding `get_transactions` as a generator means that the return value is no longer consistent - what shall we do to keep it consistent?